api.go

```diff
@@ -95,7 +95,7 @@ type LineFragmentMatch struct {
 	// Number bytes that match.
 	MatchLength int
 
-	SymbolInfo *Symbol
+	SymbolInfo Symbol
```

Comment:
why this change? For text search this will always be nil (the more common case).